javascript/npm/draco3d/draco_nodejs_example.js

```diff
@@ -82,81 +84,37 @@ function encodeMeshToFile(mesh, decoder) {
   decoderModule.destroy(ia);
   meshBuilder.AddFacesToMesh(newMesh, numFaces, indices);
 
-  // Add position data to mesh.
-  const posAttId = decoder.GetAttributeId(mesh, decoderModule.POSITION);
-  if (posAttId === -1) {
-    console.log("No position attribute found.");
-    encoderModule.destroy(newMesh);
-    encoderModule.destroy(encoder);
-    return;
-  }
-  const posAttribute = decoder.GetAttribute(mesh, posAttId);
-  const posAttributeData = new decoderModule.DracoFloat32Array();
-  decoder.GetAttributeFloatForAllPoints(
-      mesh, posAttribute, posAttributeData);
-  let vertices = new Float32Array(numVertexCoord);
-  for (let i = 0; i < numVertexCoord; i += 3) {
-    vertices[i] = posAttributeData.GetValue(i);
-    vertices[i + 1] = posAttributeData.GetValue(i + 1);
-    vertices[i + 2] = posAttributeData.GetValue(i + 2);
-  }
-  decoderModule.destroy(posAttributeData);
-  meshBuilder.AddFloatAttributeToMesh(newMesh, encoderModule.POSITION,
-                                      numPoints, 3, vertices);
-
-  const normalAttId = decoder.GetAttributeId(mesh, decoderModule.NORMAL);
-  if (normalAttId > -1) {
-    console.log("Adding normal attribute.");
-    const norAttribute = decoder.GetAttribute(mesh, normalAttId);
-    const norAttributeData = new decoderModule.DracoFloat32Array();
-    decoder.GetAttributeFloatForAllPoints(mesh, norAttribute,
-        norAttributeData);
-    const normals = new Float32Array(numVertexCoord);
-    for (let i = 0; i < numVertexCoord; i += 3) {
-      normals[i] = norAttributeData.GetValue(i);
-      normals[i + 1] = norAttributeData.GetValue(i + 1);
-      normals[i + 2] = norAttributeData.GetValue(i + 2);
-    }
-    decoderModule.destroy(norAttributeData);
-    meshBuilder.AddFloatAttributeToMesh(newMesh, encoderModule.NORMAL,
-        numPoints, 3, normals);
-  }
+  const attrs = {POSITION: 3, NORMAL: 3, COLOR: 3, TEX_COORD: 2};
+
+  Object.keys(attrs).forEach((attr) => {
+    const stride = attrs[attr];
+    const decoderAttr = decoderModule[attr];
+    const encoderAttr = decoderModule[attr];
```

Comment:
I'm not sure if it really matters but for `encoderAttr` , this should be probably taken from `encoderModule[attr]`